sceptre/cli/__init__.py

```diff
@@ -39,63 +38,35 @@
     help="The formatting style for command output.")
 @click.option("--no-colour", is_flag=True, help="Turn off output colouring.")
 @click.option(
-    "--var", multiple=True, help="A variable to template into config files.")
+    "--var", multiple=True,
+    help="A variable to replace the value of an item in config file.")
 @click.option(
     "--var-file", multiple=True, type=click.File("rb"),
-    help="A YAML file of variables to template into config files.")
+    help="A YAML file of variables to replace the values of items in config files.")
 @click.option(
-    "--ignore-dependencies", is_flag=True, help="Ignore dependencies when executing command.")
+    "--ignore-dependencies", is_flag=True,
+    help="Ignore dependencies when executing command.")
+@click.option(
+    "--merge-vars", is_flag=True, default=False,
+    help="Merge dict keys in successive vars and var files.")
```

Comment:
How about this instead?  `Merge variables from successive --vars and var files` or `Aggregate variables  from successive --vars and var files`

Reply:
ok done.